projects/mongobench-runall.yml

```diff
@@ -0,0 +1,30 @@
+---
+# Run all tests
+runner: mongobench
+params:
+  dir: mongodb-benchmarking
+  args:
+    - -threads
+    - 10
+    - -docs
+    - 100000
+    - -uri
+    - {{.MONGODB_URI}}
+    - --runAll
```

Comment:
Simple example from readme https://github.com/idealo/mongodb-benchmarking?tab=readme-ov-file#run-all-tests.
It takes about 1 min to run, do we want to tweak the parameters?